cookie-banner-rules-list.json

```diff
@@ -6997,7 +6861,19 @@
         "bloomberg.com",
         "t-online.de",
         "wetteronline.de",
-        "chip.de"
+        "chip.de",
+        "gutekueche.at",
+        "businessinsider.com",
+        "n-tv.de",
+        "bergfex.at",
+        "meczyki.pl",
+        "newsnow.co.uk",
+        "news.sky.com",
```

Comment:
I think this should be `sky.com` rather than `news.sky.com`. We don't have subdomain support yet. However, looking at the cookie banner, it's inside a `privacy-mgmt.com` frame, so we shouldn't need this domain at all.

Reply:
Removed from domains gutekueche.at, businessinsider.com, bergfex.at, meczyki.pl, news.sky.com, hs.fi, wetteronline.at, aamulehti.fi, thetimes.co.uk, idealo.de (using privacy-mgmt.com for the iframe). Thank you!